index.js

```diff
@@ -12,32 +23,66 @@ const eth = new Eth(provider);
 
 var express = require('express'),
     cors = require('cors'),
-    app = express();
+    app = express(),
+    bodyParser = require('body-parser'),
+    jsonParser = bodyParser.json();
 
 app.use(cors());
 
-app.get('/address/:address', function(req, res, next){
-  eth.getTransactionCount(address, (err, nonce) => {
-    eth.sendTransaction({
-      from: address,
-      to: req.params.address,
-      gas: 100000,
-      value: (parseFloat(process.env.AMOUNT) || 1.5) * 1e18,
-      data: '0xde5f72fd', // sha3('faucet()')
-      nonce,
-    }, (err, txID) => {
-      if (err) {
-        console.log('Request failed', err)
-        return res.status(500).json(err)
+// Store issue ids and their bounty addresses
+var issueData = {};
+
+// Receive a POST request at the address specified by an env. var.
+app.post('/address/:address', jsonParser, function(req, res, next){
```

Comment:
Why is the ':address' kept here? Doesn't look it is used anywhere with `req.params.address`